### ISSUE TITLE
Odd Highlighting On Greek and Sub/Superscripts

### ISSUE THREAD

**Issue by duncanam** [arcticicestudio/nord-vim#149]
After being able to successfully install and use Nord on vim quite nicely, I did notice a quirky thing that may or may not be intentional and I was wondering if there was a setting to turn it off or if it was indeed a bug. I am using a plugin in vim that will render some of my LaTeX expressions as Greek characters inline, and will apply sub and superscripts to them. The Nord theme seems to highlight these:
![nordissue](https://user-images.githubusercontent.com/22781288/55897055-01c15d80-5b7d-11e9-8ff4-d873d1568622.png)

Current output: highlights Greek and sub/superscript
Desired output: no highlighting of Greek and sub/superscript

Is there anything I can do about this, or is just outright unsupported (which is fine too, although it would be neat if there was a solution).

**Comment by arcticicestudio** [comment]
It is more likely that the plugin uses custom syntax keys. This would require to add support for them to Nord to make it work properly. Could you please post the name and/or the link to the GitHub repository of the plugin?

**Comment by robotdna** [comment]
@arcticicestudio Certainly, it's Vimtex, and found here:
[https://github.com/lervag/vimtex](url)

Anything else I can do to help?

**Comment by duncanam** [comment]
Well, I reached out to @gillescastel and he said to add `hi Conceal ctermbg=none` in my .vimrc and this worked for me. I no longer have the odd highlighting and everything is working as expected!

**Comment by arcticicestudio** [comment]
@duncanam Nice to hear it works fine now :tada: 
I haven't got the time to take a look into this, but the solution will also help other users facing the same problem. I've [added a note to the backlog](https://github.com/arcticicestudio/nord-vim/projects/1#card-20680649) to add it to Nord Vim's docs/troubleshooting guide :smile:

**Comment by xarthurx** [comment]
`hi Conceal ctermbg=none` doesn't work for me here.
Neovim for Windows 10.

I found some post here:
https://github.com/kaicataldo/material.vim/issues/12
for the 'material` theme, as they add some defined colour to the scheme.

Could we also do it for Nord?

**Comment by Bobby-Huggins** [comment]
For anyone else running into this issue:

`hi Conceal ctermbg=none` didn't work for me, because I am using a gui version (gVim on Windows 10).
However, `hi Conceal guibg=NONE` works, but only if it comes *after* `set background=dark`

The full theme section of my `_vimrc` is then:
```
if (has("termguicolors"))
  set termguicolors
endif
colorscheme nord
set background=dark
hi Conceal guibg=NONE
```

**Comment by nikinbaidarr** [comment]
> Well, I reached out to @gillescastel and he said to add `hi Conceal ctermbg=none` in my .vimrc and this worked for me. I no longer have the odd highlighting and everything is working as expected!

This works perfect.

**Comment by arcticicestudio** [comment]
@duncanam @xarthurx @Bobby-Huggins @nikinbaidarr Thanks for your help :+1:
The problem has been documented in #256 ⇄ #261 (⊶ f3f28b93) and will be shipped in [version `0.16.0`][1].

[1]: https://github.com/arcticicestudio/nord-vim/projects/22
